client/options.cpp

```diff
@@ -1942,44 +1945,46 @@ static struct client_option client_options[] = {
                    COC_FONT, 0, -100, 100, allfont_changed_callback),
     GEN_FONT_OPTION(gui_qt_font_default, "default_font", N_("Default font"),
                     N_("This is default font"), COC_FONT,
-                    "Sans Serif,10,-1,5,75,0,0,0,0,0",
+                    "Linux Biolinum O,12,-1,5,50,0,0,0,0,0",
```

Comment:
What happens if we have the TTF and not the OTF? I would hope for a gentle fallback/substitution...

Reply:
Then let's remove `*.ttf` from the list of files we can load

Reply:
Actually on my system it will fallback if you don't pass the O:
```
$ fc-match "Linux Biolinum"
LinBiolinum_R.otf: "Linux Biolinum O" "Regular"
```